README.md

```diff
@@ -258,4 +259,4 @@ During the installation process several projects are included (in [`./vendor`](.
 * [format-converters](https://github.com/OCR-D/format-converters) by OCR-D, [`Apache 2.0`](https://github.com/OCR-D/format-converters/blob/master/LICENSE)
 * [prima-page-converter](https://github.com/PRImA-Research-Lab/prima-page-converter/) by PRImA Research Lab , [`Apache 2.0`](https://github.com/PRImA-Research-Lab/prima-page-converter/blob/master/LICENSE)
 * [page-to-alto](https://github.com/kba/page-to-alto/) by Konstantin Baierer @kba, [`Apache 2.0`](https://github.com/OCR-D/format-converters/blob/master/LICENSE)
```

Comment:
kba/page-to-alto still has no LICENSE file. Ideally it should get one, then the link here could be updated.

Reply:
It does now.